VideoLocker/res/values/strings.xml

```diff
@@ -442,11 +442,10 @@
     <!--  Email subject line when user attempts to submit feedback about the app -->
     <string name="email_subject">Customer Feedback</string>
 
-    <!-- Message indicating videos deleted. Argument is number of videos deleted -->
-    <plurals name="deleted_videos">
-        <item quantity="one">%s video deleted</item>
-        <item quantity="other">%s videos deleted</item>
-    </plurals>
+    <!-- Message indicating one video deleted-->
+    <string name="deleted_video">{video_num} video deleted</string>
```

Comment:
I think we miscommunicated about plurals. This should still be a plural.

What we should do is get the plural string with the count and then apply a format after resolving it to one string.